### DISCUSSION TITLE
fomo奖池，推动价格回归基准价

### DISCUSSION THREAD

**Discussion by otto-001** [famland/website#81] [Ideas]
奖金池搞一个fomo奖池的规则，回归基准价之前买入的前100名，获得奖励，按越靠近回归基准价的人奖励越大。
这样大家就抢着买入，把前面的人挤出去

在回归基准价期间不能有YL的卖出和转账记录

**Comment by ginxiao** [comment]
想法不错，最好不用奖金池，因为如果发U，没有这种程度可以驱动的积累。如果用奖金池的YL发，为了拿奖金的人总要回收成本，又造成下跌踩踏。反反复复下来，会造成奖金池的FF超发而池干，都会引起系统危机。
所以此种奖金方式只能适合短线活动节日时展开。
如果要长期展开，可以发放每次的买入排名NFT，比如某时间或第几次回归基准价第几名社区守护者NFT勋章。
